### PR TITLE
Tomas' solutions for #14

### PR DESCRIPTION
**Pull request by TomasEliz**

### CODE REVIEW

File: tomas/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+let valueToCheck = '100em'
+console.log('100em not a number -->', isNaN(valueToCheck))
+
+// What is the type of NaN?
+console.log('What is the type of NaN? -->', typeof NaN)
+
+// rounding
+
+let mult = .1 * .2
+console.log('rounded sum -->', mult.toFixed(2))
```
Comment (ciamac-da) [inline tomas/index.js:17]: You forgot second part and third part of task3! 
console.log((0.6378399).toFixed(1));
console.log((1.84222).toFixed(4));

**Review by ciamac-da — CHANGES_REQUESTED** [review]

File: tomas/index.js (hunk 1 above)
Comment (ciamac-da) [inline tomas/index.js:16]: You forgot to solve task3!
console.log((12.2600000000033).toFixed(2));
console.log((0.6378399).toFixed(1));
console.log((1.84222).toFixed(4));

**Review by ciamac-da — CHANGES_REQUESTED** [review]

**Comment by TomasEliz** [comment]
uhm, damn:)
will get to it right away. Thanks!

**Review by ciamac-da — APPROVED** [review]
Good Job 👍🏻

**Pull request merged**